test/api.test.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
+
+const expect = require('chai').expect;
+require('dotenv').config()
+const {apiStatus, specstoreApiUpload, apiAuthChangeUsername, apiAuthPublicKey, apiStatusWithHeaders, apiBitstoreAuthorize, apiBitstoreInfoValidToken, apiBitstoreInfoInvalidToken} = require('../scripts/api.js')
+
+
+describe('specstore service api', function () {
+  it('specstore status works fine', async function () {
+    const url = `${process.env.SPECSTORE}/${process.env.OWNERID}/${process.env.DATAPACKAGE_NAME}/1`
+    const body = await apiStatus(url)
+    expect(body.state).to.equal('SUCCEEDED')
+  })
+  it('specstore info for latest revision works fine', async function () {
+    this.timeout(120000);
+    const url = `${process.env.SPECSTORE}/${process.env.OWNERID}/${process.env.DATAPACKAGE_NAME}/latest`
```

Comment:
I would define these constant variables once above and avoid all this `process.env.` each time I need them Eg
```
const SPECSTORE = process.env.SPECSTORE
cont OWNERID=process.env.OWNERID
etc...
```

Reply:
BTW why do we need to pass data package name or owner name as an env variable? We could set it in config instead, no?

Reply:
Good idea about not using env.variable for package name and owner name. I will define list of objects with all datasets and owners. Also, define all constant variables above `const SPECSTORE = process.env.SPECSTORE`